test/runtests.jl

```diff
@@ -15,15 +15,20 @@ testdata("static.arrow", "lhm/static.arrow")
 
 @testset "Ribasim" begin
     @safetestset "Input/Output" begin
+        using TestReports
+        recordproperty("name", "Input/Output")  # TODO To check in TeamCity
         include("io.jl")
     end
     @safetestset "Configuration" begin
         include("config.jl")
     end
+
     # @safetestset "Water allocation" begin include("alloc.jl") end
     @safetestset "Equations" begin
+        include("../utils/testdata.jl")  # to include Teamcity specific utils
         include("equations.jl")
     end
```

Comment:
The way it's set up now is that all test files that are included in a safetestset, can be run by themselves, as long as the test data is available. It'd be nice to keep that property, so that these extra lines go into the test files that need them.